### PR TITLE
Apply all arguments to analyze command.

### PR DESCRIPTION
**Pull request by bobbytables**
This is an attempt to fix: https://github.com/codeclimate/codeclimate/issues/60

### CODE REVIEW

File: spec/cc/cli/analyze_spec.rb [hunk 1]
```diff
@@ -2,6 +2,21 @@
 
 module CC::CLI
   describe Analyze do
+    describe '#initialize' do
+      it 'sets dev mode for --dev' do
+        instance = Analyze.new(['--dev'])
+
+
+        assert instance.dev_mode?, "in dev mode"
```
Comment (GordonDiggs) [inline spec/cc/cli/analyze_spec.rb]: We use the `must_` syntax - can you reformat these lines? Should be `instance.dev_mode?.must_equal true`

**Comment by pbrisbin** [comment]
This LGTM with two notes:

- We double-quote all strings (also, `%w[ ]` might be good for the spec arguments)
- Adding `#dev_mode?` only for the spec is a smell to me, but we have plans to upgrade to a real options parser soon, so I'm OK not worrying about it for now.

/cc @GordonDiggs WDYT?

**Comment by GordonDiggs** [comment]
@pbrisbin Agreed!

**Comment by bobbytables** [comment]
@pbrisbin @GordonDiggs - Style comments applied. I agree `dev_mode?` is an odd method but it allowed me quickly try this out.

A real option parser would help these problems.

**Pull request merged**